OneOfDiagnosticSuppressor/OneOfDiagnosticSuppressor.csproj

```diff
@@ -23,7 +24,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="3.11.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.9.2" />
```

Comment:
This breaks compatibility with all SDKs but the latest one.
Please use the lowest version without the problematic Http reference.